my-sms/Gemfile

```diff
@@ -0,0 +1,43 @@
+source 'https://rubygems.org'
+
+gem 'rails', '~> 3.2.0'
```

Comment:
@josh-cooper you need to run `bin/bundle update rails` after a change like that. Your `Gemfile.lock` hasn't changed: https://github.com/josh-cooper/my-sms/pull/1/files#diff-77cc97b3356eedfe96888b3cbadc69ae82664d338c3b211ca83504d209954e1eR88
https://github.com/josh-cooper/my-sms/pull/1/files#diff-77cc97b3356eedfe96888b3cbadc69ae82664d338c3b211ca83504d209954e1eR88